### ISSUE TITLE
Support for X.com links

### ISSUE THREAD

**Issue by ayancey** [SimonBrazell/privacy-redirect#438]
Twitter has almost completely migrated their domain to X.com. X.com links do not redirect through this extension over to Nitter.

**Comment by Yetangitu** [comment]
Apply this patch and run `npm run build`, copy/link/rename the resulting `.zip` to `.xpi` and install it in Firefox and it works again.

```diff
diff --git a/src/assets/javascripts/helpers/twitter.js b/src/assets/javascripts/helpers/twitter.js
index f520d76..e1fb1b0 100644
--- a/src/assets/javascripts/helpers/twitter.js
+++ b/src/assets/javascripts/helpers/twitter.js
@@ -5,8 +5,11 @@
   */
 const targets = [
   "twitter.com",
+  "x.com",
   "www.twitter.com",
+  "www.x.com",
   "mobile.twitter.com",
+  "mobile.x.com",
   "pbs.twimg.com",
   "video.twimg.com",
 ];
```

**Comment by edmael** [comment]
Hey @Yetangitu, is there a reason why this is not in the main code?

**Comment by Kobayen** [comment]
That makes the extension into an unsigned extension, and then it becomes unusable.

**Comment by Yetangitu** [comment]
> Hey @Yetangitu, is there a reason why this is not in the main code?

I guess because the developer has not implemented it yet and nobody has made a PR for this change yet. I can submit it, maybe it gets picked up. Don't hold your breath though, there are loads of unmerged PRs for this extension.

**Comment by Yetangitu** [comment]
> That makes the extension into an unsigned extension, and then it becomes unusable.

Nah, it works fine, it'll nag but since you know there reason for the nag there is no problem other than the occasional nag. You'll need to disable mandatory signature verification by setting `xpinstall.signatures.required` to `false` in `about:config` though. You still get warnings about unsigned extensions but you can run them.

![image](https://github.com/user-attachments/assets/65e949dc-847f-494c-a6fc-b03a9ca855b0)

Here's the PR: #440
